2016-09-13 17:27:41 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:41 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:42 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:42 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:43 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:43 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:44 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:44 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:45 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:45 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:46 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:46 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:47 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:47 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:48 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:48 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:49 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
2016-09-13 17:27:49 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:375)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:358)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2893)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2463)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:102)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]" [50000-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 30 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:E:/github/note-back/background-rest/db/note_back.mv.db [1.4.192/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:344)
	... 25 more
